### PR TITLE
[ADD] activity on crm for partner

### PR DESCRIPTION
**Pull request by vrenaville**
It's add activity on CRM for partner, depend on PR #155

### CODE REVIEW

File: partner_activity_report_crm/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Display crm open for this partner
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/partner-contact/issues/new?body=module:%20partner_activity_report%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (damdam-s) [inline partner_activity_report_crm/README.rst]: partner_activity_report_crm

**Comment by pedrobaeza** [comment]
Maybe this one should be on OCA/crm repo?

**Comment by rafaelbn** [comment]
Which is the criteria in OCA to decide this? This module is a report of a Partner/contact but it has dependencies with CRM.

**Comment by pedrobaeza** [comment]
The criteria is the very subjected one of "The best repo that fits to the purpose of the module". As some repos can overlap their boundaries, sometimes it's not very clear where to put. For example: _website_event_photo_, a module that you know. It's a module about events, but it corresponds to the website part, so it can be hosted on event or website projects. In this case, as we have a specialized repo for events, I would bet that a better hosting is event project, because the website project is also very crowded with other website related things. In this case, I would say the contrary, because in this case, this repo is the one very crowded, and if you think it, almost everything is partner-related, so I would bet for moving to the crm part, that isolate a bit that part of the ecosystem.

**Comment by gurneyalex** [comment]
@pedrobaeza I'd rather have all the modules linked to #155 live in the same repository as it makes maintenance of the code much easier.

**Comment by pedrobaeza** [comment]
Well, I still prefer the other option, but I won't block a PR here, because as I said, it's a matter of taste.

**Comment by rafaelbn** [comment]
OK, this means that all modules related with https://github.com/OCA/partner-contact/pull/144 purpose will be here in this report (all together)

And all will have name: partner_activity_report_\* (crm: lead, oportinities, calls; sales: sales orders; ...)

¿OK @gurneyalex @pedrobaeza ?

cc @vrenaville

**Comment by pedrobaeza** [comment]
I must insist with OCA/crm question, because I found a case where people navigating through OCA repo expect to find all modules related to CRM on the link that he/she has on the website about this area, that leads to https://github.com/OCA/crm. Having there the _partner_activity_crm_ module, he/she can read the description and decides that the module is of them interest. Installation section then tell them to install the dependency with partner-contact repo.

What do you think about this reasoning?

**Comment by rafaelbn** [comment]
@gurneyalex how much complicated makes maintenance having modules in different projects?

I agree with @pedrobaeza. It will be more intuitive to find.

Moreover, we planned to make a module for reporting partners about the events the have participate, I thought event's project would be the best project.

"The best repo that fits to the purpose of the module": Is the purpose of this module to inform Partners about their CRM activity? This means OCA/crm fits better.

**Comment by gurneyalex** [comment]
let's move then.

**Comment by gurneyalex** [comment]
@vrenaville can you move this module to OCA/crm?

**Comment by rafaelbn** [comment]
Hi @vrenaville do you want to continue with this PR? Could you please move it to OCA/crm. Let me know please

**Comment by rafaelbn** [comment]
WIP as depends on https://github.com/OCA/partner-contact/pull/155

**Comment by NL66278** [comment]
Closed as related PR closed and no activity for a long time.

**Pull request closed without merging**